funcs/func_cut.c

```diff
@@ -75,6 +75,11 @@
 		</syntax>
 		<description>
 			<para>Cut out information from a string (<replaceable>varname</replaceable>), based upon a named delimiter.</para>
+						<para>Below are examples illustrating the correct and invalid syntax.</para>
+					<example title="The 'varname' parameter could only accept a variable name, not a variable expression">
```

Comment:
Commit message looks good now.

One other thing I'm seeing here now: `could` implies past tense, change this to `can`.
Also, there is only one example, so I personally don't really think the "Below are examples illustrating the correct and invalid syntax" is needed... the comments in the example already show which is why, so this doesn't add anything.